examples/plot_held_out_enet.py

```diff
@@ -127,7 +127,7 @@
 ax.scatter(
     monitor.alphas[:, 0] / alpha_max,
     monitor.alphas[:, 1] / alpha_max,
-    s=50, cmap=cmap, c=c,
+    s=50, color=cmap(c),  zorder=len(c),
```

Comment:
zorder= high number  should be enough (it's so that it appears on top of other plotted stuff)